### PR TITLE
chore: improve redirects for org-level-billing

### PR DESCRIPTION
**Pull request by kevcodez**
- Fixes redirects on project subscription page
- Automatic redirect on project invoices page
- Adds the appropriate panel key for addon upsell, so the redirects work correctly

### CODE REVIEW

File: studio/components/interfaces/BillingV2/Subscription/Subscription.tsx [hunk 1]
```diff
@@ -24,41 +23,13 @@ const Subscription = ({}: SubscriptionProps) => {
...
-  useEffect(() => {
-    if (panel && typeof panel === 'string' && allowedValues.includes(panel)) {
-      snap.setPanelKey(panel as SUBSCRIPTION_PANEL_KEYS)
-    }
-  }, [panel])
-
   // No need to bother rendering, we'll redirect anyway
   if (isOrgBilling) return null
 
+  if (panel && typeof panel === 'string' && allowedValues.includes(panel)) {
```
Comment (joshenlim) [inline studio/components/interfaces/BillingV2/Subscription/Subscription.tsx:29]: just to check though - must this be outside of a useEffect?

Comment (kevcodez) [reply studio/components/interfaces/BillingV2/Subscription/Subscription.tsx:29]: useEffect is probably safer for this, can adjust

Comment (joshenlim) [reply studio/components/interfaces/BillingV2/Subscription/Subscription.tsx:29]: yeah if possible lets put this in a useEffect - i feel iffy about having this block re-running on each re-render

Comment (kevcodez) [reply studio/components/interfaces/BillingV2/Subscription/Subscription.tsx]: Adjusted

**Review by joshenlim — APPROVED** [review]

**Pull request merged**